utils/json-utils/src/main/scala/pl/touk/nussknacker/engine/json/swagger/SwaggerTyped.scala

```diff
@@ -132,7 +127,9 @@ object SwaggerArray {
 }
 
 object SwaggerObject {
-  import AdditionalProperties._
+  def apply(elementType: Map[PropertyName, SwaggerTyped]): SwaggerObject =
+    SwaggerObject(elementType, AdditionalPropertiesWithoutType)
```

Comment:
Why not default directly in `SwaggerObject` definition?

Reply:
Because it's case class, and we shouldn't do default value here? But I understand you want to by consistent with json schema?